source/packet.hpp

```diff
@@ -26,7 +26,7 @@ struct Packet {
     IFlow* flow;
     TimeNs sent_time;  // Note: ACK's sent time is the data packet sent time
     SizeByte
-        sent_bytes_at_origin;  // For ACK this is inherited from data packet
+        delivered_at_origin;  // For ACK this is inherited from data packet
```

Comment:
And the same thing here
```suggestion
        delivered_bytes_at_origin;  // For ACK this is inherited from data packet
```

Reply:
Renamed to `delivered_data_size_at_origin`